compiler/parser/parser.cpp

```diff
@@ -888,6 +888,10 @@ static bool dynoRealizeErrors(void) {
     if (!issuedErrors.insert(err).second) continue;
     if (fDetailedErrors) {
       chpl::Context::defaultReportError(gContext, err);
+      // Use production compiler's exit-on-error functionality for errors
+      // reported via new Dyno mechanism
+      setupDynoError(err->kind());
+      exitIfFatalErrorsEncountered();
```

Comment:
shouldn't this call be outside of the loop over errors? So other errors can be printed?

Reply:
Yes, and in fact your comment prompted me to realize it isn't necessary here at all because the production compiler already invokes `USR_STOP` (which does the same thing, but with some more printing) after calling `dynoRealizeErrors`. All we need is the `setupDynoError` portion.

Reply:
Huh, but then, why was this change necessary at all? If the bug was caused by the program not exiting on error (I think it was, it went on into another pass), why _didn't_ it exit thanks to the `USR_ERROR` outside of `dynoRealizeErrors`?

Reply:
`setupDynoError` is still needed to set the error flags (`exit_eventually` etc)